### DISCUSSION TITLE
Does anyone know what triggers autocrop.exe

### DISCUSSION THREAD

**Discussion by ghost** [staxrip/staxrip#569] [Q&A] [answered]
While testing some random filters, suddenly an autocrop.exe appeared when i dropped a video file in the main window.

How was that triggered? And how can I trigger it myself in a script?

When i dropped that same file again, no autocrop.exe was triggered...

Any idea ?

**Comment by Dendraspis** [comment]
As far as I see the only way is to call it from Crop window.  🤔

**Comment by stax76** [comment, marked as answer]
It only runs when you open a source file and the crop filter is checked and all crop values are 0.

**Reply by ghost** [reply]
Thanks, good to know !  Thanks for the answer 😃
